src/components/Activity/Email.tsx

```diff
@@ -27,11 +27,17 @@ const Email: FC<EmailActivity["properties"]> = ({
             />
             <TextBlockWithLabel
                 label="Sent by"
-                text={getFullName({ firstName: hs_email_from_firstname, lastName: hs_email_from_lastname })}
+                text={getFullName({
+                    firstName: hs_email_from_firstname,
+                    lastName: hs_email_from_lastname,
+                } as Owner)}
```

Comment:
This value doesn't match the type because of missing fields. If `getFullName` only requires those to then you can use `Pick<...>` to ensure it matches up correctly.

Reply:
fixed types in `getFullName`